app/src/main/java/org/mozilla/fenix/components/toolbar/ToolbarIntegration.kt

```diff
@@ -67,28 +66,17 @@ class ToolbarIntegration(
         }
     }
 
-    private val toolbarFeature: ToolbarFeature = ToolbarFeature(
+    private val toolbarPresenter: ToolbarPresenter = ToolbarPresenter(
```

Comment:
Please do not break apart the feature :(

We are working on follow-up features that will land in this feature. If there's something that doesn't work right please communicate that (as an AC issue ideally). Otherwise we end up with some frankenstein monster where Fenix has essentially forked AC. :)

Reply:
Just noticed that this is only for the editing mode and the display mode still uses `ToolbarFeature`. Now I'm less concerned. :)